quickwit-cluster/src/utils.rs

```diff
@@ -0,0 +1,31 @@
+//  Quickwit
+//  Copyright (C) 2021 Quickwit Inc.
+//
+//  Quickwit is offered under the AGPL v3.0 and as commercial software.
+//  For commercial licensing, contact us at hello@quickwit.io.
+//
+//  AGPL:
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Affero General Public License as
+//  published by the Free Software Foundation, either version 3 of the
+//  License, or (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Affero General Public License for more details.
+//
+//  You should have received a copy of the GNU Affero General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pub mod rendezvous_hasher;
+
+use std::net::SocketAddr;
+
+const GRPC_PORT_INC: u16 = 1;
```

Comment:
I think we can move that const somewhere in quickwit-searcher

Reply:
Moved the const to `quickwit-search` package.
Please see: https://github.com/quickwit-inc/quickwit/pull/143/commits/81bbf716ecdd01c85b8110dde4418a3c0c64e2e8